README.md

```diff
@@ -34,7 +31,10 @@ If you would like to contribute, please fork the repository, introduce your chan
 
 ### License
 
-The code in this repository is licensed under the terms of the the Apache License 2.0.
+The code in this repository is licensed as follows:
+
+- all crates under `bin` directory are licensed under the terms of the GNU GPL version 3
+- all the rest crates are licensed under the terms of Apache License 2.0.
```

Comment:
Nit: `all the rest crates` -> `rest of the crates`

Reply:
done